### DISCUSSION TITLE
Offline ERD and View to Table conversion

### DISCUSSION THREAD

**Discussion by lamprey13** [dbeaver/dbeaver#21870] [General]
Hi,

I had two questions about using dbeaver:
1. I can create an ERD from an existing database and save it to a n ERD file. When I open that ERD dbeaver wants to connect to the database that it was created from and if it can't it offers a dialog to pick a new data source. But, is there a way to just open an ERD without the need to connect to a database? I just want to view the ERD and make changes.
2. I have a DB with a bunch of Views and I'd like to load them into dbeaver Diagram and then be able to "convert" them to a tables so I can add in the PKs and relationships (foreign keys) so I can show how they connect. Is there any functionality like that?

Thanks!

**Comment by Seb33300** [comment]
I have the same issue, did you find any way to generate offline ERD?

**Comment by lamprey13** [comment]
Unfortunately, no.
